helpers/mailer.js

```diff
@@ -21,14 +21,16 @@ var transporter = nodemailer.createTransport({
 
 module.exports = {
 
-    readHTMLTemplate: function (templateName, callback) {
-        fs.readFile(__dirname + "/../templates/" + templateName, {encoding: 'utf-8'}, function (err, html) {
-            if (err) {
-                callback(err, null);
-            }
-            else {
-                callback(null, html);
-            }
+    readHTMLTemplate: function (templateName) {
+        return new Promise((resolve, reject) => {
+            fs.readFile(__dirname + "/../templates/" + templateName, {encoding: 'utf-8'}, function (err, html) {
```

Comment:
Can we omit the overall path all together so that it could be passed from the callee ?
Just to not have a string hardcoded and other dynamic. It's perhaps better to keep it overall dynamic.

Reply:
Correct, sure we can do that.